### DISCUSSION TITLE
Hotkey for changing floppy? [expediate diskmapper ops]

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1116] [Q&A] [answered]
In the case where you have an Amiga title comprising 2 (or more) floppy disk images, same which are loaded into the diskmapper slots, I find that it really interrupts the visual continuity of the emulation, when you have to hit F12 -> diskmapper panel -> select next disk image (usually to load into DF0 ) -> hit F12/click resume ...to continue the emulation.

In these cases where the diskmapper is required, would it be possible to add a Hotkey, which advances to the next disk in the diskmapper list and loads that disk into DF0?

If this were a Hotkey, one could map same to a controller button or keyboard? This would be really handy...

Any chance Amiberry could do something like this?

TIA

**Comment by midwan** [comment, marked as answer]
Amiberry already supports the same shortcuts as WinUAE for the Disk Swapper.
Additionally, you can use Custom Controls to assign any of the supported events to be triggered on a controller button press.

**Comment by giantclambake** [comment]
Ah, OK...thanks --- is that missing from the wiki documentation, or did I miss it?
